Lib/test/test_configparser.py

```diff
@@ -980,6 +980,50 @@ def test_set_nonstring_types(self):
         self.assertRaises(TypeError, cf.set, "sect", 123, "invalid opt name!")
         self.assertRaises(TypeError, cf.add_section, 123)
 
+    def test_str(self):
+        self.maxDiff = None
+        cf = self.config_class(allow_no_value=True, delimiters=('=',), strict=True)
+        cf.add_section("sect1")
+        cf.add_section("sect2")
```

Comment:
No need for that many sections right? we can have a smaller test for `__str__` I think.

Reply:
Done